dev/com.ibm.ws.security.acme/resources/OSGI-INF/l10n/metatype.properties

```diff
@@ -16,8 +16,50 @@
 
 # ACME configuration
 
-#configParm1=Configuration parameter 1 for ACME
-#configParm1.desc=Description for parameter 1
+#directoryURI=ACME server directory URI
+#directoryURI.desc=The URI to the ACME server's directory object.
 
-#configParm2=Configuration parameter 2 for ACME
-#configParm2.desc=Description for parameter 2
+#domain=Domain name
+#domain.desc=A domain name to request a certificate for.
+
+#validFor=Valid for
+#validFor.desc=The duration of time the certificate signing request will request the certificate to be valid for. The default is defined by the CA server.
```

Comment:
**change**
The duration of time the certificate signing request will request the certificate to be valid for. The default is defined by the CA server.
**to**
The duration of time that the certificate signing request specifies for the certificate to be valid. The default is defined by the CA server.

Reply:
Made the requested changes.